src/ember-app.js

```diff
@@ -66,7 +68,23 @@ class EmberApp {
 
     // Ensure that the dist files can be evaluated and the `Ember.Application`
     // instance created.
-    this.buildApp();
+    this.buildSandboxQueue(options.maxSandboxQueue);
+  }
+
+  /**
+   * @private
+   *
+   * unction to build queue of sandboxes which is later leveraged if application is using buildSandboxPerRequest
```

Comment:
(spelling)
_function to build queue of sandboxes which is later leveraged if the application is using buildSandboxPerRequest_

Reply:
Thanks for catching this.